k8s/namespaces/em/em-hrs-ingestor/prod-0.yaml

```diff
@@ -6,4 +6,7 @@ spec:
   values:
     job:
       schedule: "0 21-23 * * *"
-      # Run every hour during the evening 9pm through 7am
+      environment:
```

Comment:
these should really be added to a patch for prod rather than cluster specific, you can then reference both patches in your per cluster config

i.e. 
```
./../../namespaces/em/em-hrs-ingestor/prod.yaml
./../../namespaces/em/em-hrs-ingestor/prod-0.yaml
```

Reply:
thanks tim, changes applied